src/project/qml/MuseScore/Project/internal/MeasuresSettings.qml

```diff
@@ -46,15 +46,15 @@ FlatButton {
 
         property string pickupMessage: {
             if (withPickupMeasure.checked) {
-                return qsTrc("project", "Pickup: ") +
+                return qsTrc("project", "Pickup:") + " " +
                         model.pickupTimeSignature.numerator + "/" + model.pickupTimeSignature.denominator
             }
 
             return qsTrc("project", "No pickup")
         }
 
         font: ui.theme.largeBodyFont
-        text: model.measureCount + qsTrc("project", " measures, \n") + pickupMessage
+        text: model.measureCount + qsTrc("project", " measures.") + "\n" + pickupMessage
```

Comment:
Please explain the purpose of this change (I mean replacing ',' with '.')

Reply:
Potential next text starts with an uppercase letter.